src/components/wallet/PageTransactions.vue

```diff
@@ -19,13 +19,24 @@
         />
       </div>
       <br />
+      <TmDataMsg icon="archive">
+        <div slot="title">
+          Past Chain Transactions
+        </div>
+        <div slot="subtitle">
+          Transactions from past chains are not displayed here. If you are
+          looking forward to accessing those,
+          <a href="mailto:contact@lunie.io">write us</a>!
```

Comment:
this should open intercom, not email anymore

Reply:
What if they are on desktop i.e. not mobile? It doesn't work 😆 

Did you test this before changing it? xD